recipes/replxx/all/conanfile.py

```diff
@@ -10,10 +11,10 @@ class ReplxxConan(ConanFile):
     """
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/AmokHuginnsson/replxx"
-    topics = ("conan", "readline", "libedit", "UTF-8")
+    topics = ("readline", "libedit", "UTF-8")
     license = "BSD-3-Clause"
     exports_sources = ["CMakeLists.txt"]
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
```

Comment:
Are the `cmake_find_package` and `cmake_find_package_multi` generators needed at all?
This package doesn't have any requirements.
```suggestion
    generators = "cmake"
```

Reply:
You are right. Thank you.
I delete this.

Reply:
Thank you.
I misunderstand the meaning of cmake_find_package.
I will fix it.

Reply:
No problem.
When the port to conan 2.0 happens, we will have to go through every recipe and replace `cmake` with `CMakeToolchain`/`CMakeDeps` anyway.